test/release/examples/primers/chplvis/prog2.chpl

```diff
@@ -0,0 +1,44 @@
+// Example 2 of use of VisualDebug module and chplvis tool.
```

Comment:
Part of my idea here was to have the verbose part in the html since it is very hard to show graphics in code.   That let the code be shorter in the html.   It was also my intent that appears to have not been communicated well, that the "users" would read the html in one window and compile/execute and run chplvis in another window.     If the html ends up as part of the web site, then I'd think there shouldn't be any programs for chplvis in the /examples/primers directory, possibly just a pointer to the web site.   (The document would link to the example programs and the user would get those programs from the web site.)  If there are any files in /examples/primers, I'd expect all of them to be there.   That is why I did a subdirectory for chplvis.   I'm speaking partly from my point of view of recently needing to read the primers to refresh my memory about Chapel and to learn about new features I hadn't used before or weren't there last time I used Chapel.